### DISCUSSION TITLE
Save options like "Group" & "Sort" per user

### DISCUSSION THREAD

**Discussion by MeesvanStraten** [community/community#8680] [Projects and Issues]
Great tool 👍 .

When changing filters like the "group by status" or new fields are added to the table, it would be nice that this is saved. Each time I open the project board the filters are reset. This way each user could have their own default view of the project board with the information they require.

Keep it up 🥇 .

**Comment by Bukama** [comment]
Yes please, especially that the sorting is always set back to highest issue id is annoying, cause one has to change it to "last updated" every time to see latest changes.
